pages/listings/[id].tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Nitpick: is this needed anymore? looks like it's left over from debugging :stuck_out_tongue: